src/inference/src/ie_core.cpp

```diff
@@ -1113,6 +1123,18 @@ class CoreImpl : public ie::ICore, public std::enable_shared_from_this<ie::ICore
     void SetConfigForPlugins(const std::map<std::string, std::string>& configMap, const std::string& deviceName) {
         auto config = configMap;
 
+        for (auto& item : config) {
+            if (item.first == ov::force_tbb_terminate.name()) {
```

Comment:
Why is force_tbb_terminate one property of pluign instead of OV runtime?

Reply:
This property should take effect for all plugins that involved tbb threads.